src/Rotations.jl

```diff
@@ -1,5 +1,47 @@
+# we generate code in this module, so precompile where possible
+ VERSION >= v"0.4.0-dev+6521" && __precompile__(true)
```

Comment:
I don't know... I've removed the check


On 9 May 2016 at 17:39, Chris Foster <notifications@github.com> wrote:

> In src/Rotations.jl
> <https://github.com/FugroRoames/Rotations.jl/pull/2#discussion_r62459166>:
>
> > @@ -1,5 +1,47 @@
> > +# we generate code in this module, so precompile where possible
> > + VERSION >= v"0.4.0-dev+6521" && __precompile__(true)
>
> The VERSION check is probably not required here - we don't support 0.3,
> right?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/FugroRoames/Rotations.jl/pull/2/files/fb3a7882257b93afed97cfaa9bf56669bbf410dc#r62459166>
>